lib/Alien/Build/Util.pm

```diff
@@ -76,7 +76,26 @@ sub _mirror
         { unlink "$dst" }
         my $target = readlink "$src";
         Alien::Build->log("ln -s $target $dst") if $opt->{verbose};
+        if (path($target)->is_relative) {
+          my $nativesymlink =
+                   (($^O eq "msys" && defined $ENV{MSYS} && $ENV{MSYS} eq "winsymlinks:nativestrict")
+                || ($^O eq "cygwin" && defined $ENV{CYGWIN} && $ENV{CYGWIN} eq "winsymlinks:nativestrict"));
+          if (!$src->parent->child($target)->exists) {
+            if ($nativesymlink) {
+              # NOTE: On linux, it is OK to create broken symlinks, but it is not allowed on 
+              #   windows MSYS2/Cygwin when nativestrict is used.
+              die "cannot create native symlink to nonexistent file $target on $^O";
+            }
+          }
+          if ($nativesymlink) {
+            $dst->parent->child($target)->touchpath;
+          }
+        }
+        my $curdir = Path::Tiny->cwd;
```

Comment:
For AB I usually use `File::chdir` to avoid clutter of tracking temporary directory changes.

Reply:
Good idea, I will update the code to use `File::chdir` instead.

Reply:
```suggestion
```

line can be removed now.